pages/contacts.mdx

```diff
@@ -0,0 +1,32 @@
+## Sage Bionetworks
+
+**Contact us regarding:** annotations, uploading data to Synapse, data management, bioinformatics anaylsis
+
+| Name | Company | Job Title | Contact |
+| :-----------: | :-----------: | :-----------: | :-----------: |
+| Anh Nguyet Vu | Sage Bionetworks | Bioinformatics Engineer | [Schedule Meeting Here](https://www.example.com) |
+| Elvira Mitraka | Sage Bionetworks | Bioinformatics Engineer |  [Schedule Meeting Here](https://www.example.com) |
+| Christina Conrad | Sage Bionetworks | Biomedical Data Manager |  [Schedule Meeting Here](https://www.example.com) |
+
+## MKSCC Team 
+
+**Contact us regarding:** ..............
+
+| Name | Company | Job Title | Contact |
+| :-----------: | :-----------: | :-----------: | :-----------: |
+| Header | Sage Bionetworks | Header | Title |
+| Header | Sage Bionetworks | Header | Title |
+| Header | Sage Bionetworks | Header | Title |
+| Header | Sage Bionetworks | Header | Title |
+
+
+## Gray Foundation Team
+
+> :bulb: **Contact us regarding:** .....................
```

Comment:
This is also pretty optional, bc I feel like the only reasons that make sense are "press releases" and "potential collaboration"? -- and we need to see if any of the teams are interested in this/want to volunteer their official lab emails.